test/test_tests_scan.py

```diff
@@ -402,6 +402,49 @@ def test_amqp_sha_pr(self, mock_queue, _mock_strftime):
             }
         }
 
+    # mock time for predictable test name
+    @unittest.mock.patch("time.strftime", return_value="20240102-030405")
+    @unittest.mock.patch("task.distributed_queue.DistributedQueue")
+    def test_amqp_sha_pr_cross_project(self, mock_queue, _mock_strftime):
```

Comment:
I wonder if I could convince you to add this as a pytest-style test.  We're going to want to convert everything at some point anyway...

You're probably interested in the monkeypatch fixture, if you do that.